tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -146,7 +146,7 @@ stages:
             name: Default
           e2e_pool_definition:
             vmImage: 'ubuntu-20.04'
-          environment: PROFILE="-Dflink.hadoop.version=2.10.2 -Dscala-2.12 -Djdk11 -Djdk17"
+          environment: PROFILE="-Dflink.hadoop.version=2.10.2 -Dscala-2.12 -Djdk11 -Djdk17 -Pjava17-target"
```

Comment:
Why do we need both `-Djdk11` and `-Djdk17` here?

Reply:
These variables are used to filter version-specific code in the shell scripts.